niceman/interface/diff.py

```diff
@@ -34,6 +34,13 @@ def __init__(self, cls):
         self.cls = cls
 
 
+def _make_plural(s):
+    """Poor man 'plural' version for now"""
+    if s.endswith('repository'):
+        return s.replace('repository', 'repositories')
+    else:
+        return s + 's'
```

Comment:
This is currently only used in one place.  Do you plan to use it elsewhere?  If not it doesn't seem worth having a separate function.